nofos/nofos/views.py

```diff
@@ -1090,6 +1092,59 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class NofoUploadCoverImageView(BaseNofoEditView):
```

Comment:
Just an observation: I created the `uploads` app because I was thinking image management would be a big thing but it seems like it was not really true.